kore/app/exec/Main.hs

```diff
@@ -453,6 +626,18 @@ mainWithOptions execOptions = do
       | otherwise =
         koreRun execOptions
 
+    writeInReportDirectory :: FilePath -> IO ()
+    writeInReportDirectory tempDirectory = do
+        when . isJust . toReport . bugReport
+            <*> writeOptionsAndKoreFiles tempDirectory $ execOptions
+        Foldable.forM_ (outputFileName execOptions)
+            $ flip copyFile (tempDirectory <> "/outputFile.kore")
+
+    archiveDirectoryReport :: FilePath -> FilePath -> IO ()
+    archiveDirectoryReport tempDirectory dest = do
+        files <- listDirectory tempDirectory
+        createTarGz (dest <> ".tar.gz") tempDirectory files
```

Comment:
`listDirectory` seems to be used every time `createTarGz` is called; can we extract a helper function?